raw.go

```diff
@@ -31,3 +31,32 @@ type RawSpan struct {
 	// The span's associated baggage.
 	Baggage map[string]string // initialized on first use
 }
+
+// SpanDetail summarizes the identifying fields of a Span, for
+// diagnostic and implementation-specific purposes.
+type SpanDetail struct {
```

Comment:
I could weigh in on this debate, but I have another idea entirely... how about creating an interface like so:

```
type BasicSpan {
    opentracing.Span

    OperationName() string
    Start() time.Time
    Context() Context
}
```

... then the ultimate user of the basictracer-derived span would do something like

```
basicSpan := otSpan.(basictracer.BasicSpan)
fmt.Println("I can read whatever I want!", basicSpan.Operation())
...
```

Just to make sure I wasn't misremembering how Go works, I wrote up this example: http://play.golang.org/p/1KllYjHi2_